bedrock/test.js

```diff
@@ -1,7 +0,0 @@
-/*
```

Comment:
This file did not get moved into `test` and therefore the mocha tests cannot be run.  This file needs to be resurrected and the `test` script in `package.json` should be updated to reflect the new location.

@dlongley do we want our convention to be to keep the mocha and protractor tests in separate folders as we are doing elsewhere?